charts/karpenter/templates/100-config-logging.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: config-logging
   namespace: {{ .Release.Namespace }}
   labels:
-    app.kubernetes.io/part-of: karpenter
```

Comment:
IIUC this is required by our runtime framework to discover the logging configuration.

Reply:
helper have : `app.kubernetes.io/part-of: {{ template "karpenter.name" . }}`
Is it ok for your runtime framework ? or i must change to `app.kubernetes.io/part-of: karpenter` @ellistarn

Reply:
I would use:
> app.kubernetes.io/part-of: karpenter

We shouldn't be parameterizing this value.